projects/go-lib/src/lib/components/go-toast/go-toast.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="go-toast">
+  <div class="go-toast-strip" [ngClass]="statusClasses()"></div>
```

Comment:
We could probably sprinkle in some more `.block__element`s in this file, but this makes sense to me.